src/components/DpCard/DpCard.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+import DpCard from './DpCard.vue'
+import DpLoading from '../DpLoading/DpLoading.vue'
+
+const meta: Meta<typeof DpCard> = {
+    component: DpCard,
+    title: "Components/Card",
+    argTypes: {},
+    render: (args) => ({
+        components: {
+            DpCard,
+            DpLoading
+        },
+        setup() {
+            return { args }
+        },
+        template: `<dp-card v-bind="args">
+          <dp-loading class="u-mt-0_5" />
```

Comment:
Do we have to show a loading component here as example? Is this the typical content of a Card?

Reply:
No this is not typical content
Done in https://github.com/demos-europe/demosplan-ui/pull/472/commits/f6547c7be93fc6ed2a61fa515711c7ff86a12e25